src/services/logs/index.ts

```diff
@@ -0,0 +1,15 @@
+import { LogsService } from './types';
+import { HttpInstance } from '../../http/types';
+import httpClient from '../http-client';
+import { LOGS_BASE } from '../../constants';
+import access from './access';
+
+export const logsService = (httpWithAuth: HttpInstance): LogsService => {
+  const logsClient = httpClient({
+    basePath: LOGS_BASE,
+  });
+
+  return {
+    access: access(logsClient, httpWithAuth),
```

Comment:
I added +1 level of abstraction here, so we can add more than just access logs in the future if another collection is added to that service.